repos.yaml

```diff
@@ -61,6 +61,11 @@
   fork: 'knative-automation/homebrew-client'
   channel: 'cli'
   assignees: knative/client-wg-leads
+- name: 'knative/func'
+  meta-organization: 'knative'
+  fork: 'knative-automation/func'
```

Comment:
@knative-sandbox/productivity-leads does this fork need to be explicitly created or does knobots do that for us?

Reply:
I can't recall off the top of my head but I wouldnt hold on this i mean if it is not created you can always create it later right?

Reply:
Agreed. But I do want to hold until the move actually happens, since this PR removes the entry for `knative-sandbox/kn-plugin-func`. @salaboy maybe this should be done as two separate PRs?

Reply:
If knobots is broken for a day, at most people won't get a few automated PRs, and they'll need to wait for another day. Not the end of the world...